mungegithub/mungers/stale-green-ci.go

```diff
@@ -105,15 +117,26 @@ func (s *StaleGreenCI) Munge(obj *github.MungeObject) {
 				glog.Errorf("Failed to write retrigger old test comment")
 				return
 			}
-			ok := obj.WaitForPending(requiredContexts)
```

Comment:
If we munged the same PR again before the context switches to pending we would make a duplicate comment in that case.